delfin/drivers/huawei/oceanstor/rest_client.py

```diff
@@ -368,6 +368,68 @@ def get_all_shares(self):
 
         return cifs + nfs + ftps
 
+    def get_all_mapping_views(self):
+        url = "/mappingview"
```

Comment:
Is this url (others as well) specific to oceanstor or common across devices or just internal?

Reply:
These url path, are specific to OceanStor backend

Reply:
ok